lib/redbird/plug/session/redis.ex

```diff
@@ -61,3 +63,19 @@ defmodule Plug.Session.REDIS do
     end
   end
 end
+
+defmodule Redbird.RedisError do
+  defexception [:message]
+
+  def raise do
+    raise __MODULE__
+  end
+
+  def exception(_) do
+    %__MODULE__{message: base_message()}
```

Comment:
This was really outdated, not sure how I managed to no push the new version. The current implementation does include the redis error.